.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "${workspaceFolder}/cmd/mattermost/main.go",
+            "cwd": "${workspaceFolder}",
+            "env": {},
+            "args": [],
+            "buildFlags": "-ldflags '-X \"github.com/mattermost/mattermost-server/v6/model.BuildNumber=debug\" -X \"github.com/mattermost/mattermost-server/v6/model.BuildBoards=true\" -X \"github.com/mattermost/mattermost-server/v6/model.MockCWS=false\"'"
```

Comment:
@wiggin77 Should `BuildBoards=true` be set?

Reply:
Probably not.  It means Boards cannot be debugged with this config, but it won't work anyway because Boards webapp won't be available.  We'll have to come up with a more complicated config for Boards (later).

Reply:
Ok, I'll remove this for now then.